install.sh

```
@@ -507,6 +501,93 @@ locate_package_dir() {
+
+# 4. Check progress
+genie agent history <agent-name>
+
+# 5. Shut down
+genie agent kill <agent-id>
+```
+</GENIE_CLI>
+ORCHESTRATION_HEREDOC
+
+    success "Orchestration prompt written to $dest"
+}
```

Shellcheck (0.11.0)

[info] This function is never invoked. Check usage (or ignored if invoked indirectly).

(SC2329)